src/NServiceBus.RabbitMQ.Tests/ConnectionString/ConnectionStringParserTests.cs

```diff
@@ -12,19 +11,18 @@ public class ConnectionStringParserTests
             "virtualHost=Copa;username=Copa;host=192.168.1.1:1234;password=abc_xyz;port=12345;requestedHeartbeat=3;" +
             "retryDelay=01:02:03;useTls=true;certPath=/path/to/client/keycert.p12;certPassPhrase=abc123";
 
-        SettingsHolder settings;
+        string endpointName;
 
         [SetUp]
         public void Setup()
         {
-            settings = new SettingsHolder();
-            settings.Set("NServiceBus.Routing.EndpointName", "endpoint");
+            endpointName= "NServiceBus.Routing.EndpointName";
```

Comment:
This should be set to "endpoint" to match the previous behavior.

Instead, `endpointName` can become a const string, and we can drop the setup method entirely.